config/locales/en.yml

```diff
@@ -35,7 +35,7 @@ en:
   feedback: "Feedback"
   issue_contact: "Please contact the SUCCESS Center at (843) 792-8300 or success@musc.edu for assistance with this process or with any questions you may have."
   portal_title: "SPARC Request Portal"
```

Comment:
I can undo the change. However, the original text is too long, it doesn't fit the button - see attached screenshot. Any other suggestions?
![create an account](https://cloud.githubusercontent.com/assets/7648699/15508665/f34d896e-2195-11e6-9edc-1d9d137c369d.jpg)